dockerfiles/cpubuilder_ubuntu_jammy_ghr_x86_64.Dockerfile

```diff
@@ -4,17 +4,25 @@ FROM docker.io/myoung34/github-runner:ubuntu-jammy
 ######## Basic apt packages ########
 RUN apt update && \
     apt install -y \
-        wget python3.11-dev python3-pip git unzip curl gnupg2 lsb-release && \
-    update-alternatives --install /usr/local/bin/python python /usr/bin/python3.11 3 && \
-    update-alternatives --install /usr/local/bin/python3 python3 /usr/bin/python3.11 3 && \
+        wget git unzip curl gnupg2 lsb-release && \
     apt install -y \
         ccache clang-14 lld-14 libssl-dev ninja-build libxml2-dev llvm-dev pkg-config \
-        libcapstone-dev libtbb-dev libzstd-dev && \
-    apt clean && rm -rf /var/lib/apt/lists/*
+        libcapstone-dev libtbb-dev libzstd-dev
+# Python
+RUN apt install -y python3.11-dev python3.11-venv python3-pip && \
+    update-alternatives --install /usr/local/bin/python python /usr/bin/python3.11 3 && \
+    update-alternatives --install /usr/local/bin/python3 python3 /usr/bin/python3.11 3
+# Toolchains and build deps
+RUN apt install -y \
+        wget git unzip curl gnupg2 lsb-release \
```

Comment:
```suggestion
```
Those are already installed as part of the basic apt packages. Either remove here (preferred) or above.

Reply:
Oops, good catch.